CLA.md

```diff
@@ -0,0 +1,85 @@
+# ZIO Contributor License Agreement
+
+Thank you for your interest in contributing to the ZIO open source project.
+
+This contributor agreement ("Agreement") describes the terms and conditions under which you may Submit a Contribution to Us. By Submitting a Contribution to Us, you accept the terms and conditions in the Agreement. If you do not accept the terms and conditions in the Agreement, you must not Submit any Contribution to Us.
+
+This is a legally binding document, so please read it carefully before accepting the terms and conditions. If you accept this Agreement, the then-current version of this Agreement shall apply each time you Submit a Contribution. The Agreement may cover more than one software project managed by Us.
+
+## 1. Definitions
+
+"We" or "Us" means Ziverge, Inc., and its duly appointed and authorized representatives.
```

Comment:
What is `Ziverge Inc`?

Reply:
@jdegoes's company.

Reply:
Does that mean all the repos under `zio` now belong to that company?

Reply:
The repositories continue to maintained by the ZIO organization moderated by the steering committee and are open source subject to the Apache license. Contributors also retain copyright in their own contributions. This grants Ziverge a license to the contributions.

Reply:
Okay, but Isn't it the case that any company had a license to use any of the libraries in the zio ecosystem without requiring explicit permission/license to use them since they are open sourced? Why does Ziverge need to have this custom license? Also for transparency it would be good know who is on the steering committee as well. I don't think I can see that information on the microsite anywhere.

Reply:
As we get closer to 1.0 release we want to make sure we have this buttoned down so that companies that adopt ZIO can know that there aren't any intellectual property issues there. 

Note that the CLA has already been in place for the main ZIO repository (though hasn't previously been added to all ZIO repositories individually).

The information on the steering committee is [here](https://zio.dev/docs/about/about_coc).